api/registrations/views.py

```diff
@@ -268,33 +275,17 @@ def get_queryset(self):
         return queryset
 
 
-class RegistrationChildrenList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, RegistrationMixin):
+class RegistrationChildrenList(BaseChildrenList, generics.ListAPIView, ListFilterMixin, RegistrationMixin):
```

Comment:
```suggestion
class RegistrationChildrenList(BaseChildrenList, generics.ListAPIView, NodesFilterMixin, RegistrationMixin):
```
Not caused by this change, but since this is a list of nodes it should use `NodesFilterMixin`. I happened to find a 502 error if you try to filter children by `parent` or `root`:
https://api.staging.osf.io/v2/registrations/xbeur/children/?filter[parent]=xbeur

It's not super consequential right now, but fixing might avoid confusion if we add things to `NodesFilterMixin`.